src/main/java/io/sentry/samples/example/Application.java

```diff
@@ -97,21 +88,12 @@ public String HandledError() {
 		try {
 			int example = 1/0;
 		} catch (Exception e) {
+			logger.error("caught exception", e);
```

Comment:
no error was getting collected, unless you include logger.error here.

Reply:
No error was reported because the exception is handled.